_includes/extensions/writing-repeater.html

```diff
@@ -202,25 +195,22 @@ <h3 id="writing-extensions-repeater-dataAttr">Data Attributes</h3>
 		<tr>
 			<td>data-container</td>
 			<td>"true"</td>
-			<td>If you add this value to an element within a returned <code>item</code>, the element will become the parent
-				container for any nested renderer within that renderer object.
+			<td>If you add this value to an element within the repeater canvas in the <code>before</code> function,
+				that element will become the <code>container</code> that is passed to the <code>renderItem</code> function.
 			</td>
 		</tr>
 		<tr>
 			<td>data-infinite</td>
 			<td>"true"</td>
-			<td>If infinite scrolling is enabled, the element with this attribute becomes the infinite
-				scrolling container.
-			</td>
+			<td>If infinite scrolling is enabled, the element with this attribute becomes the infinite scrolling container.</td>
 		</tr>
 		<tr>
 			<td>data-preserve</td>
 			<td>"deep" or "shallow"</td>
-			<td>The repeater will preserve an element containing this attribute upon clearing the repeater of data. This clear
-				occurs during each <code>dataSource</code> call. Set the value to <code>"deep"</code>
-				to preserve the element and all children. Set the value to
-				<code>"shallow"</code> to preserve just the element and remove any
-				child nodes not marked with <code>data-preserve</code>.
+			<td>The repeater will preserve an element containing this attribute upon clearing the repeater of content.
+				This clear occurs during each <code>dataSource</code> call. Set the value to <code>"deep"</code>
+				to preserve the element and all children. Set the value to <code>"shallow"</code> to preserve just the
+				element and remove any child nodes not marked with <code>data-preserve</code>.
```

Comment:
could this result in a number of elements marked as data-preserve? could a config option be added `data-preserver="whitelist"` or `"blacklist"` so that maybe you keep everything but the things you mark to remove? 

just a thought